tidy3d/components/eme/data/sim_data.py

```diff
@@ -39,13 +39,23 @@ class EMESimulationData(AbstractYeeGridSimulationData):
         None, title="S Matrix", description="Scattering matrix of the EME simulation."
     )
 
-    port_modes: Optional[EMEModeSolverData] = pd.Field(
+    port_modes_raw: Optional[EMEModeSolverData] = pd.Field(
         None,
         title="Port Modes",
         description="Modes associated with the two ports of the EME device. "
-        "The scattering matrix is expressed in this basis.",
+        "The scattering matrix is expressed in this basis. "
+        "Note: these modes are not symmetry expanded; use 'port_modes' instead.",
     )
 
+    @property
```

Comment:
`@cached_property` would be more in line with how things work in the mode solver with `data` and `data_raw`, and also could speed up things in some use cases.